bin/spryker-sdk.sh

```diff
@@ -24,23 +24,23 @@ myIp='host.docker.internal'
 
 if [ "$(uname)" == "Linux" ] && [ "$(uname -a | grep -c -v Microsoft | sed 's/^ *//')" -eq 1 ]; then
   myIp=$(ip route get 1 | sed 's/^.*src \([^ ]*\).*$/\1/;q')
-  return 0
 fi
 
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
+export EXECUTABLE_FILE_PATH=$(realpath "$0")
 
 case $MODE in
 "debug")
   echo "Ensure mutagen is running by executing: mutagen compose -f docker-compose.yml -f docker-compose.dev.yml up -d"
   export SPRYKER_XDEBUG_HOST_IP=${myIp}
   export PHP_IDE_CONFIG=serverName=spryker-sdk
-  docker-compose -f "${SDK_DIR}/docker-compose.yml" -f "${SDK_DIR}/../docker-compose.dev.yml" run --rm -e XDEBUG_SESSION=1 -e APP_ENV=dev spryker-sdk "$ARGUMENTS"
```

Comment:
I think the command should use docker-compose._debug_.yml same as build/bin/spryker-sdk.sh